CHANGELOG.md

```diff
@@ -1,9 +1,150 @@
+- [#668](https://github.com/TokTok/c-toxcore/pull/668) Make Ping_Array a module-private type.
+- [#667](https://github.com/TokTok/c-toxcore/pull/667) pkg-config .pc files: added .private versions of Libs and Required
+- [#665](https://github.com/TokTok/c-toxcore/pull/665) Remove useless if statement
+- [#662](https://github.com/TokTok/c-toxcore/pull/662) Move Networking_Core struct into the .c file.
+- [#661](https://github.com/TokTok/c-toxcore/pull/661) Disable asan, since it seems to break on travis.
+- [#660](https://github.com/TokTok/c-toxcore/pull/660) Increase test retries to 10 (basically infinite).
+- [#659](https://github.com/TokTok/c-toxcore/pull/659) Fix formatting in some C files.
+- [#658](https://github.com/TokTok/c-toxcore/pull/658) Call freeaddrinfo on error paths in net_getipport.
+- [#657](https://github.com/TokTok/c-toxcore/pull/657) Zero-initialise stack-allocated objects in hstox driver.
+- [#656](https://github.com/TokTok/c-toxcore/pull/656) Fix file descriptor leak in hstox test.
+- [#652](https://github.com/TokTok/c-toxcore/pull/652) Add support for building the monolith test on android.
+- [#650](https://github.com/TokTok/c-toxcore/pull/650) Remove deprecated ToxDNS
+- [#648](https://github.com/TokTok/c-toxcore/pull/648) Make hstox compile on FreeBSD
+- [#624](https://github.com/TokTok/c-toxcore/pull/624) Update rpm spec and use variables in cmake instead of hardcoded paths
+- [#616](https://github.com/TokTok/c-toxcore/pull/616) Add projects link to Readme.
+- [#613](https://github.com/TokTok/c-toxcore/pull/613) Fix travis
+- [#605](https://github.com/TokTok/c-toxcore/pull/605) Fix OS X Travis.
+- [#598](https://github.com/TokTok/c-toxcore/pull/598) Fix typos in docs
+- [#578](https://github.com/TokTok/c-toxcore/pull/578) Split toxav_bit_rate_set() into two functions to hold the maximum bitrates libvpx supports
+- [#477](https://github.com/TokTok/c-toxcore/pull/477) Update install instructions to use CMake
+- [#465](https://github.com/TokTok/c-toxcore/pull/465) Add Alpine linux Dockerfile in addition to the existing Debian one
+- [#442](https://github.com/TokTok/c-toxcore/pull/442) Generate only one large library "libtoxcore".
+- [#334](https://github.com/TokTok/c-toxcore/pull/334) Change toxencryptsave API to never overwrite pass keys.
+
+### Closed issues:
+
+- [#704](https://github.com/TokTok/c-toxcore/issues/704) Add CORRECTION support to group chats
+- [#620](https://github.com/TokTok/c-toxcore/issues/620) WIP: Video bug found!
+- [#606](https://github.com/TokTok/c-toxcore/issues/606) groupId is int whereas friendId is uint32_t, reason?
+- [#572](https://github.com/TokTok/c-toxcore/issues/572) int32_t may be not large enough as a argument for video_bit_rate of vp8/9 codec
+- [#566](https://github.com/TokTok/c-toxcore/issues/566) LAYER #: modules for static linking - build issue
+- [#42](https://github.com/TokTok/c-toxcore/issues/42) Remove ToxDNS and related stuff from toxcore
+
+## v0.1.12
```

Comment:
0.1.12 was never released i think

Reply:
Yeah, there's no 0.1.12 tagged, but this is autogenerated from our milestones. I guess those two items should be moved to the 0.2.0 milestone. I'll have to regenerate this after the persistent conferences are merged, so I'll sort it out then.

Reply:
Sounds like the persistent conferences will not make it in 0.2.0, so you don't have to wait on them being merged to sort this out.

Reply:
Here is the citation for the persistent conferences not making it in: https://github.com/TokTok/c-toxcore/issues/810.